configs/AARCH64_zynqmp_verified.cmake

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env -S cmake -P
+#
+# Copyright 2022, Proofcraft Pty Ltd
```

Comment:
do we care about updating this to 2024 (same in other config)?

Reply:
There are no significant changes apart from the platform string, so I left it on the old date.